internal/ordering_test.go

```diff
@@ -492,7 +492,7 @@ policies:
 			wantFile: "",
 			wantErr:  "extraDependencies may not be set in policy one manifest[0] because orderManifests is set",
 		},
-		"orderManifests and consolidateManifests is false": {
+		"orderManifests is true and consolidateManifests is unset": {
```

Comment:
Nit: can we add a tiny bit more context to these tests?
```suggestion
		"orderManifests is true in policyDefaults, and consolidateManifests is unset": {
```